logs.txt

```diff
@@ -0,0 +1,4 @@
+Nov 26, 2020 9:31:29 PM models.LogToFileModel log
```

Comment:
Do not commit the logs.txt file (this file should remain blank or else logs will already be present in the file)

Reply:
i do not change this part, it was from the previous delivery